cfgov/mega_menu/tests/test_frontend_conversion.py

```diff
@@ -109,11 +110,16 @@ def test_converted_format_uses_basic_python_types(self):
     def test_conversion_database_queries(self):
         self.menu.refresh_from_db()
 
-        # We expect to see two queries here:
+        # We expect to see three queries here:
```

Comment:
What is the third query that gets added here?

Reply:
See comment [above](https://github.com/cfpb/cfgov-refresh/pull/5826#discussion_r453725163); I think this is due to changes in Site handling in Wagtail 2.9.

Reply:
Is this still related to the `Site` handling changes? Are there still three queries instead of two?